CodeJam.Main/ConnectionStrings/ConnectionStringBase.cs

```diff
@@ -260,5 +222,45 @@ public bool EquivalentTo(ConnectionStringBase other)
 		/// <inheritdoc />
 		public override string ToString() => _wrapper.ToString();
 		#endregion
+
+		#region KeywordDescriptor class
+		/// <summary>
+		/// Descriptor for connection string keyword
+		/// </summary>
+		[PublicAPI]
+		protected class KeywordDescriptor
```

Comment:
As far as I can remember we usually do place nested classes at the beginning of the file.

There were no explicit guideline for this so I'm curious: should we keep existing style or it's ok to place nested types where it seems to be more natural?

Reply:
Nested classes at the beginning of the file make it very difficult to read the code, so I would still leave it that way.